src/flows/pipes/TestFlux/view.tsx

```diff
@@ -91,6 +91,10 @@ const TestFlux: FC<PipeProp> = ({Context}) => {
         emptyText="This cell will visualize results from uploaded CSVs"
         emptyIcon={IconFont.BarChart}
         toggleVisibilityEnabled={false}
+        height={data.panelHeight}
+        onUpdateHeight={panelHeight => update({panelHeight})}
+        visibility={data.panelVisibility}
```

Comment:
why not make the height and visibility component-based like you did here:

https://github.com/influxdata/ui/pull/117/files#diff-f62f29439ccd6eea91c6b82414e62b0cR82-R84

Reply:
because i totally forgot about that component. good catch!

Reply:
oh nope, i found out why. it's connected to the display height of the panel itself, not just the results of the panel. it makes more sense on the visualization component this was cloned from, but the user should be able to resize the panel contents, and have that persist across reloads

Reply:
That makes sense. I’m assuming it’s the same reason as other view file 👍🏽